f5xc/ce/gcp/outputs.tf

```diff
@@ -3,7 +3,7 @@ output "ce" {
     master-0 = {
       node    = module.node[0].ce["master-0"]
       config  = module.config[0].ce["master-0"]
-      network = module.network[0].ce["master-0"]
+      network = local.create_network ? module.network[0].ce["master-0"] : null
```

Comment:
Same here... if we are using "existing" subnet(s), the network is not being created, should this be an empty string (`""`) or `null` like this... probably should be consistent with the other one ;)